components/server/src/workspace/workspace-starter.ts

```diff
@@ -921,16 +921,8 @@ export class WorkspaceStarter {
                 featureFlags.push("workspace_psi");
             }
 
-            // if the workspace has been created in an organization, we need to use the organization's attribution ID
-            let usageAttributionId = AttributionId.createFromOrganizationId(workspace.organizationId);
-            if (!usageAttributionId) {
-                if (!user.additionalData?.isMigratedToTeamOnlyAttribution) {
-                    usageAttributionId = await this.userService.getWorkspaceUsageAttributionId(user);
-                } else {
-                    if (usageAttributionId === undefined) {
-                        throw new Error("No usage attribution ID found");
-                    }
-                }
+            if (!workspace.organizationId) {
+                throw new Error("No usage attribution ID found");
```

Comment:
```suggestion
                throw new Error("No usage attribution ID found - could not derive attribution from workspace organization");
```

Reply:
This in general looks just like a safety. What are the reasons we'd not have an Org ID on a workspace? Is this older data?